### ISSUE TITLE
ApplicationOnlyCredentials

### ISSUE THREAD

**Issue by dosborn** [linvi/tweetinvi#97]
I am trying to create a Filtered Stream using application only credentials and it always hits the stopped event with a null reference exception. If I switch it to use an access token as well then it works fine.

I'm using the below code:

var credentials = Auth.SetApplicationOnlyCredentials("[CONSUMER KEY]", "[CONSUMER SECRET]", true);

//this works fine
//var credentials = new TwitterCredentials("[CONSUMER KEY]", "[CONSUMER SECRET]", "[ACCESS TOKEN]", "[ACCESS TOKEN SECRET]");

var stream = Tweetinvi.Stream.CreateFilteredStream(credentials);
//add tracks
 stream.StartStreamMatchingAllConditionsAsync();

Should this functionality be working?

**Comment by linvi** [comment]
As mentioned in the Twitter [doc](https://dev.twitter.com/oauth/application-only), you cannot access a stream endpoint with Application Only credentials.

```
With Application-only authentication you won’t be able to:

Post tweets or other resources;
Connect in Streaming endpoints;
Search for users;
Use any geo endpoint;
Access DMs or account credentials;
```

I will put that has an improvement so that the error raised informs the developer that this is not possible.

**Comment by linvi** [comment]
This issue was already fixed in 0.9.10.x.
